core/src/test/java/org/infinispan/commands/PutMapCommandStressTest.java

```diff
@@ -59,6 +60,11 @@ protected void createCacheManagers() throws Throwable {
       if (biasAcquisition != null) {
          builderUsed.clustering().biasAcquisition(biasAcquisition);
       }
+      // UNCOMMENT the following to test put command under topology change with a store
+//      builderUsed.persistence()
+//            .addStore(DummyInMemoryStoreConfigurationBuilder.class)
+//               .shared(true)
+//               .storeName(PutMapCommandStressTest.class.toString());
```

Comment:
We don't have a job for stress tests in CI anyway, but we should have a parameter tho enable the store.

Reply:
Yeah, time constraints :)